server/api/region.go

```diff
@@ -557,6 +559,71 @@ func (h *regionsHandler) GetTopSize(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+// @Tags region
+// @Summary Accelerate regions schedule in given range
+// @Accept json
+// @Param body body object true "json params"
+// @Produce json
+// @Param limit query integer false "Limit count" default(16)
+// @Produce json
+// @Success 200 {string} string "The regions in given range have been improve schedule priority"
+// @Failure 400 {string} string "The input is invalid."
+// @Router /regions/keys/accelerate-schedule [post]
+func (h *regionsHandler) AccelerateRegionsScheduleInRange(w http.ResponseWriter, r *http.Request) {
+	rc := getCluster(r.Context())
+	var input map[string]interface{}
+	if err := apiutil.ReadJSONRespondError(h.rd, w, r.Body, &input); err != nil {
+		return
+	}
+	parseKey := func(name string, input map[string]interface{}) (string, error) {
+		k, ok := input[name]
+		if !ok {
+			return "", fmt.Errorf("missing %s", name)
+		}
+		key, ok := k.(string)
+		if !ok {
+			return "", fmt.Errorf("bad format %s", name)
+		}
+		return key, nil
+	}
+
+	startKey, err := parseKey("start_key", input)
+	if err != nil {
+		h.rd.JSON(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	endKey, err := parseKey("end_key", input)
+	if err != nil {
+		h.rd.JSON(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	limit := defaultRegionLimit
```

Comment:
16 seems to small

Reply:
@disksing  Do you have any advice? What about 256?

Reply:
Sounds good to me